pkg/client/apiutil/restmapper.go

```diff
@@ -194,13 +208,28 @@ func (m *mapper) addKnownGroupAndReload(groupName string, versions ...string) er
 		return fmt.Errorf("failed to get API group resources: %w", err)
```

Comment:
I think we now don't have the lock anymore when calling fetchGroupVersionResourcesLocked in l.206. Probably we just have to move it up from below

Reply:
Its moved up